### PR TITLE
Fixed several compile warnings.

### PR DESCRIPTION
**Pull request by UniQP**

### CODE REVIEW

File: t/if-then-else.c [hunk 1]
```diff
@@ -45,9 +45,9 @@ static void out(void *ctx, const char *p, size_t n) {
 }
 
 int main(void) {
-    unibi_var_t var_dyn   [26] = {0};
-    unibi_var_t var_static[26] = {0};
-    unibi_var_t param[9] = {0};
+    unibi_var_t var_dyn   [26] = {{0}};
```
Comment (rofl0r) [inline t/if-then-else.c]: the warning about missing braces in the universal initializer is bogus and was removed in recent gcc versions
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119

**Comment by UniQP** [comment]
The {{0}}-style is also used in unibilium.c and tools/unibi-put.c. Thus, the patch improves consistency.

**Comment by mauke** [comment]
The `static` thing makes sense, but `{{0}}` assumes too much about the internal representation of `unibi_var_t`. (I know I documented it myself, but I changed my mind.) It's OK if `unibilium.c` does it because it is the implementation.

**Comment by UniQP** [comment]
I removed the corresponding commit.

**Comment by mauke** [comment]
I believe this is already fixed.

**Pull request closed without merging**